src/zenml/zen_server/deploy/helm/templates/server-deployment.yaml

```diff
@@ -45,6 +60,13 @@ spec:
             - name: zenml-config
               mountPath: /zenml/.zenconfig
           env:
+            - name: REQUESTS_CA_BUNDLE
+              value: /etc/ssl/certs/ca-certificates.crt
+            - name: SSL_CERT_FILE
+              value: /etc/ssl/certs/ca-certificates.crt
+            # Force Node.js to use system CAs
```

Comment:
how to make it use system CAs?

Reply:
@mazay so to clarify: the `pymysql` SQLAlchemy driver that we use in ZenML will respect the `REQUESTS_CA_BUNDLE` value. This means that we can set `REQUESTS_CA_BUNDLE` to point to the CA certificate bundle that we prepare by appending custom CA certs to the system one provided by the container image in `/etc/ssl/certs/ca-certificates.crt`. We wouldn't need to configure `zenml.database.sslCa` anymore (i.e. we won't need to point the MySQL URL to the CA certificate file).